core/class_db.cpp

```diff
@@ -480,6 +480,8 @@ uint64_t ClassDB::get_api_hash(APIType p_api) {
 			for (List<StringName>::Element *F = snames.front(); F; F = F->next()) {
 
 				PropertySetGet *psg = t->property_setget.getptr(F->get());
+				if (!psg)
+					continue;
```

Comment:
If that's a hypothetical case that doesn't actually happen, this should probably be `ERR_CONTINUE()`, so that we know that we're getting a broken hash. CC @neikeq

Reply:
I would use `CRASH_COND` here, but [the code above](https://github.com/godotengine/godot/blob/017b224a87064a3d333ba63809a0f12cfe2c4412/core/class_db.cpp#L378) uses `ERR_FAIL_COND_V` so that may be better for consistency.

Reply:
Should be `ERR_FAIL_COND_V(!psg, 0)` then @qarmin.